xtask/src/main.rs

```diff
@@ -37,8 +38,9 @@ impl BuildParams {
             if matches.contains_id("release") { Profile::Release } else { Profile::Debug };
         let verbose = matches.contains_id("verbose");
         let arch = Arch::Aarch64;
+        let wait_for_gdb = matches.try_contains_id("gdb").unwrap_or(false);
```

Comment:
Why do you need this instead of `contains_id`?

Reply:
I'm wondering if we should move to a struct for defining the subcommands now that they're diverging: https://docs.rs/clap/3.2.14/clap/_derive/_tutorial/index.html

Reply:
That could be cool; I certainly wouldn't be opposed. I suggest we land this more or less as-is now, and then perhaps circle back to it later.